hadoop-ozone/dist/src/main/smoketest/compatibility/om.robot

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       Test om compatibility
+Library             BuiltIn
+Resource            ../lib/os.robot
+Test Timeout        5 minutes
+
+*** Test Cases ***
+Picks up command line options
+    Pass Execution If    '%{HDFS_OM_OPTS}' == ''    Command-line option required for process check
```

Comment:
Shouldn't we rename this envvar also to OZONE_OM_OPTS?

Reply:
No, these tests intentionaly use old names to verify compatibility with old scripts.  Some of the variables (eg. `HDFS_OM_OPTS` and `HDFS_STORAGECONTAINERMANAGER_OPTS`) were already deprecated before this change, but I changed the function that deprecates them.

https://github.com/apache/ozone/blob/dfd2aaf7fff4815e906e7201cee1739ad9776d97/hadoop-ozone/dist/src/shell/ozone/ozone#L160